### DISCUSSION TITLE
Wazuh app on SO

### DISCUSSION THREAD

**Discussion by rusqq** [Security-Onion-Solutions/securityonion#6636] [Unsupported Versions] [answered]
Hello,

how to connect to wazuh app? I believe the app is available on port 55000 but it doesnt work for me.

**Comment by weslambert** [comment, marked as answer]
We do not currently support the Wazuh app. See: https://github.com/Security-Onion-Solutions/securityonion/issues/237. Unless you are referring to the API, which can be accessed after allowing the appropriate IPs via so-allow.
